validation/helpers.py

```diff
@@ -206,9 +206,45 @@ def get_distance_with_translations(word):
         matches = get_translations(results)
 
         suggestions[word] = {
+            "transcription": word,
             "med": suggestions[word],
             "matches": matches,
             "len": len(matches) if len(matches) == 1 else len(matches) + 1,
         }
 
     return suggestions
+
+
+def perfect_match(word, suggestions):
+    # Checks for exactly one entry with either
+    # MED = 0 or an exact match for spelling
+    # Returns none if no matches found, or if multiple matches found
```

Comment:
Nitpick: convert comments like these to docstrings so that automatic documentation discovery tools can easily pull it out.

```suggestion
    """
    Checks for exactly one entry with either
    MED = 0 or an exact match for spelling
    Returns none if no matches found, or if multiple matches found
    """
```